src/contexts/AppContext.tsx

```diff
@@ -1,4 +1,4 @@
-import { type Project, type Fundamental } from "@/interfaces";
+import { type Project, type Fundamental, type EthIntro } from "@/interfaces";
```

Comment:
@wolovim why are we importing `EthIntro` here? it's not being used, right? :thinking: 

```ts
type EthIntro
```

Reply:
oh weird, i thought i deleted all these. i can add a quick PR to remove them first.